Patch-Source: https://sources.debian.org/data/main/libg/libgit-repository-perl/1.325-3/debian/patches/git-2.40.0.patch
--
Description: Skip subtest when git version >= 2.40.0
 'git var GIT_EDITOR' behaviour was changed in git 2.40.0:
 https://github.com/git/git/commit/2ad150e35e52705a97f18bee08f34e00a6557d6f
 so skip the test which expect the old error messages if git is new enough.
Origin: vendor
Bug: https://github.com/book/Git-Repository/pull/23
Forwarded: https://github.com/book/Git-Repository/pull/23
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2023-03-22

--- a/t/20-simple.t
+++ b/t/20-simple.t
@@ -85,6 +85,9 @@
     skip "this test does not work with msysgit on Win32", 2
         if $^O eq 'MSWin32';
 
+    skip "'git var GIT_EDITOR' behaviour was changed in git 2.40.0, and we have $version", 2
+        if Git::Repository->version_ge('2.40.0');
+
     ok( !eval { $r->run( var => 'GIT_EDITOR' ); 1; }, 'git var GIT_EDITOR' );
     like(
         $@,
